subsystems/sidecar/lib/http.js

```diff
@@ -164,7 +164,11 @@ module.exports = class Http extends ReadyResource {
         }
         app.warmup({ protocol, batch })
       }
-      const stream = await bundle.streamFrom(link.filename)
+
+      const meta = await this.entry(link.filename)
+      if (meta === null) return
```

Comment:
this needs to also end the http request with a 404

Reply:
I did as elsewhere in the fail and now throw a ERR_HTTP_NOT_FOUND error, but there's a bug unrelated to this PR: there's no special case for this error in the catch-handler, so it returns a 500 code instead of a 404: https://github.com/holepunchto/pear/blob/d984e3bd9839e5444fbc3d8a485a1377cdf96d7b/subsystems/sidecar/lib/http.js#L46-L54